test/smoke/appServers/global-resources/FixedRateSampling_ApplicationInsights.json

```diff
@@ -1,12 +1,8 @@
 {
-  "instrumentationSettings": {
-    "connectionString": "InstrumentationKey=00000000-0000-0000-0000-0FEEDDADBEEF;IngestionEndpoint=http://fakeingestion:60606/",
-    "preview": {
-      "sampling": {
-        "fixedRate": {
-          "percentage": 50
-        }
-      }
+  "connectionString": "InstrumentationKey=00000000-0000-0000-0000-0FEEDDADBEEF;IngestionEndpoint=http://fakeingestion:60606/",
+  "sampling": {
+    "traceIdRatio": {
+      "ratio": 0.5
```

Comment:
should this be probability instead of ratio?

Reply:
oops, yes and also `traceIdRatio` should be `traceIdBased`, i will fix (unfortunately this smoke test doesn't run which is why this didn't fail the build)